code/modules/mob/living/carbon/human/death.dm

```diff
@@ -12,11 +12,11 @@
 	if(buckled)
 		buckled.unbuckle_mob(src, force = TRUE)
 
-	Stun(100, TRUE, TRUE)//hold them still as they get deleted so they don't fuck up the animation
 	notransform = TRUE
```

Comment:
Why are we setting notransform?
Would it not be better to call ``death()`` on the mob at the start of the proc?

Reply:
because it fucks with the animation if they die immediately, rather than standing normally and fading away, they slowly fall over while fading away